packages/processor/processor.js

```diff
@@ -69,6 +69,8 @@ class Processor {
             // eslint-disable-next-line no-empty-function
             () => {};
 
+        this._loadFile = options.loadFile;
```

Comment:
To avoid having to have conditional `if(this._loadFile)` checks everywhere provide a default value that is a function that calls a `util.promisify`'d version of `fs.readFile()`.

Will require a small change to how `.file()` works so it awaits on the call to `.loadFile()`, but that's super-doable since it's already promise-returning.